packages/govuk-frontend/src/govuk/components/header/_index.scss

```
@@ -273,7 +296,7 @@
 
       // Increase top/bottom padding
       padding-top: govuk-spacing(3);
-      padding-bottom: govuk-spacing(3);
+      padding-bottom: govuk-spacing(3) - $govuk-header-rebrand-logo-bottom-margin;
```

```suggestion
      padding-bottom: govuk-spacing(2) + $govuk-header-rebrand-logo-bottom-margin;
```

To get to calculated `padding-bottom: 12px;`